series/series_linux_test.go

```diff
@@ -192,22 +181,46 @@ BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
 	"trusty",
 	"",
 }, {
+	`NAME="Arch Linux"
+ID=arch
+PRETTY_NAME="Arch Linux"
+ANSI_COLOR="0;36"
+HOME_URL="https://www.archlinux.org/"
+SUPPORT_URL="https://bbs.archlinux.org/"
+BUG_REPORT_URL="https://bugs.archlinux.org/"
+`,
+	"genericlinux",
 	"",
-	"unknown",
-	"OS release file is missing ID",
 }, {
-	`NAME="CentOS Linux"
-ID="centos"
+	`NAME=Fedora
+VERSION="24 (Twenty Four)"
+ID=fedora
+VERSION_ID=24
+PRETTY_NAME="Fedora 24 (Twenty Four)"
+CPE_NAME="cpe:/o:fedoraproject:fedora:24"
+HOME_URL="https://fedoraproject.org/"
+BUG_REPORT_URL="https://bugzilla.redhat.com/"
 `,
-	"unknown",
-	"OS release file is missing VERSION_ID",
+	"genericlinux",
+	"",
 }, {
 	`NAME="SuSE Linux"
 ID="SuSE"
 VERSION_ID="12"
 `,
-	"unknown",
+	"genericlinux",
 	"",
+}, {
+
+	"",
+	"unknown",
+	"OS release file is missing ID",
+}, {
+	`NAME="CentOS Linux"
```

Comment:
That test is checking what happens when the os-release file says "CentOS" but there's no information to determine the CentOS series (hence the "could not determine series" error)